internal/sponge/item/item.go

```diff
@@ -33,6 +34,14 @@ func Upsert(context interface{}, db *db.DB, item *Item) error {
 		return err
 	}
 
+	// If CreatedAt is not set, set it. This allows for CreatedAt to be set for data import.
+	if item.CreatedAt.IsZero() {
```

Comment:
I'd rather see this being added above where we check to see if the ID is set. If the ID is not set, then we obviously need to set the `CreatedAt` timestamp.